app/src/androidTest/java/com/github/h3lp3rs/h3lp/messaging/ConversationTest.kt

```diff
@@ -5,9 +5,9 @@
 //import com.github.h3lp3rs.h3lp.database.Databases.Companion.setDatabase
 //import com.github.h3lp3rs.h3lp.database.Databases.MESSAGES
 //import com.github.h3lp3rs.h3lp.database.MockDatabase
-//import com.github.h3lp3rs.h3lp.messaging.Conversation
-//import com.github.h3lp3rs.h3lp.messaging.Message
-//import com.github.h3lp3rs.h3lp.messaging.Messenger.HELPER
+//import com.github.h3lp3rs.h3lp.model.messaging.Conversation
+//import com.github.h3lp3rs.h3lp.model.messaging.Message
```

Comment:
Why is this class commented? Might as well remove it completely for the release.

Reply:
Indeed